homeassistant/components/airthings/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "id": "ID",
+          "secret": "Secret",
+          "description": "Login at https://dashboard.airthings.com/integrations/api-integration to find your credentials"
```

Comment:
We should inject URL via a placeholder and not have it be part of the translation text.